deployment/ansible/roles/nyc-trees.tiler/templates/upstart-nyc-trees-tiler.conf.j2

```diff
@@ -0,0 +1,11 @@
+description	"nyc-trees-tiler"
+
+start on ({{ "vagrant-mounted" if developing_or_testing else "local-filesystems and net-device-up IFACE!=lo" }})
```

Comment:
This template conditional doesn't evaluate properly. If I change it to something that should trigger the `else`, it keeps `vagrant-mounted`. See the fix I used for the application Upstart script in https://github.com/azavea/nyc-trees/commit/cfb558b1e72eb0cd1d7341127a2bd94fc5afad1b.